code/ui/blocks/src/components/ArgsTable/ArgsTable.tsx

```diff
@@ -107,6 +107,24 @@ export const TableWrapper = styled.table<{
       marginLeft: inAddonPanel ? 0 : 1,
       marginRight: inAddonPanel ? 0 : 1,
 
+      [`tr:first-child${ignoreSsrWarning}`]: {
+        [`td:first-child${ignoreSsrWarning}, th:first-child${ignoreSsrWarning}`]: {
+          borderTopLeftRadius: inAddonPanel ? 0 : theme.appBorderRadius,
+        },
+        [`td:last-child${ignoreSsrWarning}, th:last-child${ignoreSsrWarning}`]: {
+          borderTopRightRadius: inAddonPanel ? 0 : theme.appBorderRadius,
+        },
+      },
+
+      [`tr:last-child${ignoreSsrWarning}`]: {
+        [`td:first-child${ignoreSsrWarning}, th:first-child${ignoreSsrWarning}`]: {
+          borderBottomLeftRadius: inAddonPanel ? 0 : theme.appBorderRadius,
+        },
+        [`td:last-child${ignoreSsrWarning}, th:last-child${ignoreSsrWarning}`]: {
+          borderBottomRightRadius: inAddonPanel ? 0 : theme.appBorderRadius,
+        },
+      },
+
```

Comment:
What's happened here? Where did all this code go/come from?

Reply:
Investigating...

Reply:
huh, that's weird :D I'm pretty sure that I didn't do it... perhaps a wrong merge?

Reply:
Seems like some kind of merge issue, will fix.

Reply:
Fixed!